--- src/cazary.es6	2016-12-09 04:13:00.000000000 +0900
+++ src/cazary-legacy.es6	2016-12-09 04:13:00.000000000 +0900
@@ -1,4 +1,4 @@
-/*! Cazary (jQuery 1.7+) - JavaScript WYSIWYG editor (https://github.com/shimataro/cazary) */
+/*! Cazary (jQuery 1.2.4+) - JavaScript WYSIWYG editor (https://github.com/shimataro/cazary) */
 (function(factory)
 {
 	if(typeof module === "object" && typeof module.exports === "object")
@@ -712,11 +712,11 @@
 			{
 				// window events
 				$(document)
-					.on("click", function()
+					.bind("click", function()
 					{
 						destroyAllPanels();
 					})
-					.on("keydown", function(event)
+					.bind("keydown", function(event)
 					{
 						if(event.keyCode === KEYCODE_ESCAPE)
 						{
@@ -819,23 +819,23 @@
 
 					// editor events
 					$(editor.contentDocument)
-						.on("select", function()
+						.bind("select", function()
 						{
 							_updateCommandStatus();
 						})
-						.on("mouseup", function()
+						.bind("mouseup", function()
 						{
 							destroyAllPanels();
 							_updateCommandStatus();
 						})
-						.on("keydown", function(event)
+						.bind("keydown", function(event)
 						{
 							if(event.keyCode === KEYCODE_ESCAPE)
 							{
 								destroyAllPanels();
 							}
 						})
-						.on("keyup paste", function()
+						.bind("keyup paste", function()
 						{
 							window.setTimeout(function()
 							{
@@ -845,12 +845,12 @@
 						});
 
 					$(editor.contentWindow)
-						.on("focus", function()
+						.bind("focus", function()
 						{
 							destroyAllPanels();
 							_updateCommandStatus();
 						})
-						.on("blur", function()
+						.bind("blur", function()
 						{
 							// update original element when focus is out
 							$origin.val(editor.value());
@@ -858,7 +858,7 @@
 
 					// cancel other handler when command is disabled
 					$cazary
-						.on("click", "ul.cazary-commands-list li", function(event)
+						.find("ul.cazary-commands-list li").bind("click", function(event)
 						{
 							const $target = $(this);
 							if($target.hasClass("cazary-disabled"))
@@ -870,7 +870,7 @@
 
 					// toggle RTE <-> HTML mode
 					$cazary_command_source
-						.on("click", function()
+						.bind("click", function()
 						{
 							_toggleMode();
 							_setFocus();
@@ -881,7 +881,7 @@
 					{
 						const commandName = this.toLowerCase();
 						$cazary
-							.on("click", `.cazary-command-${commandName}`, function()
+							.find(`.cazary-command-${commandName}`).bind("click", function()
 							{
 								// execute command
 								_execCommand(commandName);
@@ -891,7 +891,7 @@
 					{
 						const commandName = this.toLowerCase();
 						$cazary
-							.on("click", `.cazary-command-${commandName}`, function()
+							.find(`.cazary-command-${commandName}`).bind("click", function()
 							{
 								// open panel
 								const $target = $(this);
@@ -1009,7 +1009,7 @@
 						{
 							// set click event to "li"
 							$panel
-								.on("click", "li", function()
+								.find("li").bind("click", function()
 								{
 									// execute command
 									const $target = $(this);
@@ -1030,7 +1030,7 @@
 								left: `${offset.left}px`,
 								top : `${offset.top}px`,
 							})
-							.on("click", function()
+							.bind("click", function()
 							{
 								// stop bubbling
 								return false;
@@ -1209,10 +1209,11 @@
 									)
 								);
 
-						return $panel
-							.on("submit", "form", onsubmit)
-							.on("click", ".cazary-widget-submit", onsubmit)
-							.on("keydown paste", ".cazary-widget-insertimage-url", onupdate);
+						$panel.find("form").bind("submit", onsubmit);
+						$panel.find(".cazary-widget-submit").bind("click", onsubmit);
+						$panel.find(".cazary-widget-insertimage-url").bind("keydown paste", onupdate);
+
+						return $panel;
 
 						function onsubmit()
 						{
@@ -1316,10 +1317,10 @@
 									)
 								);
 
-						return $panel
-							.on("submit", "form", onsubmit)
-							.on("click", ".cazary-widget-submit", onsubmit)
-							.on("keydown paste", ".cazary-widget-createlink-url", onupdate);
+						$panel.find("form").bind("submit", onsubmit);
+						$panel.find(".cazary-widget-submit").bind("click", onsubmit);
+						$panel.find(".cazary-widget-createlink-url").bind("keydown paste", onupdate);
+						return $panel;
 
 						function onsubmit()
 						{
